pabi_security/xlsx_template/xlsx_template_wizard.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+      # Assign Security
+      <act_window id="action_assign_security_export_xlsx"
+          name="Export as Excel (.xlsx)"
+          res_model="export.xlsx.template"
+          src_model="pabi.security"
+          view_mode="form"
+          target="new"
+          context="{'lang': 'en_US'}"/>
```

Comment:
Please use lang = 'th_TH' (in all excel export, we normally want to display content in Thai language when possible)